sources/Interop/Windows/um/wincon/Windows.cs

```diff
@@ -0,0 +1,372 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/wincon.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Windows
+    {
+        [DllImport("kernel32", EntryPoint = "AllocConsole", ExactSpelling = true)]
```

Comment:
There are `SetLastError` entries missing form these. @nathan-alden-sr actually wrote a program to help find these, but I can't seem to find it anymore.